doc/articles/using-uno-ui.md

```diff
@@ -387,14 +387,46 @@ Adding a custom font is done through the use of WebFonts, using a data-URI:
 
 ```css
 @font-face {
-  font-family: "Symbols";
+  font-family: "Roboto";
```

Comment:
Why did you change the font family ?

Reply:
Ah! Originally with good intentions - I wanted to show the custom font matching there directly, so I wanted to use well-known font name. But then I split it to make it clearer... And didn't roll this one back 😀